Mail/Utils/View+Extension.swift

```diff
@@ -23,12 +23,15 @@ import SwiftUI
 
 struct DeviceRotationViewModifier: ViewModifier {
     let action: (UIInterfaceOrientation?) -> Void
+    @State var lastOrientation = UIApplication.shared.mainSceneKeyWindow?.windowScene?.interfaceOrientation
```

Comment:
I just have a question, could we use a binding instead of an internal state to handle rotation ?

Reply:
We checked together quickly this morning with @Ambrdctr it was not helping or easier to read.